pages/13.contributing/06.Writer/docs.md

```diff
@@ -32,6 +32,126 @@ Please note that we do check to ensure that articles are not copied from other l
+
+Community blog authors keep their copyright but the Mautic Project and Acquia also have the right to reuse the work.  If someone does not want to give the Mautic Project and Acquia that right, it should be discussed with the Project Lead.
+
+##### Article Submission Guide
+
+_Note: This is based on [Joomla project](https://magazine.joomla.org/all/item/71-article-submission-guide "Joomla project")_
+
+###### Our audience
+
+The international Mautic community includes everyone, from first time users to experienced developers.  We also talk to marketers that may be using other platforms, but that might want to migrate to Mautic once they get to know our tool a little better.  All submissions should be provided in US English.
+
+###### Word count
+
+For most subjects, we are looking for articles consisting of approximately 400-650 words, with 500 words as a good target.
+
+More in-depth articles will be considered depending on the subject matter.  If you are interested in writing about a subject that is extremely long, consider breaking it up into a series of multiple articles.  If this is the case, please submit a summary of your subject.
+
+###### Submission deadline
+
+Please submit articles at least seven calendar days before the expected publish date.  This allows time for editorial review, amendments and publishing.  Also, please be available to review any questions or suggestions that you may get before publishing your article.
+
+###### What we look for
+
+Successful article submissions will have the following in common:
+
+  - The subject matter is interesting and relevant to the Mautic Community audience and marketers, and has a positive angle.
+  - The articles present sufficient ‘stand alone’ information for the reader.  This means that a high level of experience or knowledge is not required to understand the article.  Including educational/explanatory links to other resources is encouraged!
+  - Authors are members of the Mautic community, e.g.  users, commenters, forum members, developers, documenters, etc.
+
+###### What to avoid
+
+  - IMPORTANT: All article must abide by our __Content Guidelines__;
+  - Articles that do not have a direct connection to Mautic;
+  - Self-promotional articles (example: an article about a plugin or service written by the developer themself, or an employee).  Even if the plugin or service in question is free, the article will not be published;
+  - Articles that draw attention to issues/problems/concerns.  Mautic has other channels for that, but that isn't the purpose of the Community Blog;
+  - Articles that are too advertorial in style – rather than blatantly plug the writer’s own business, articles should employ the 'attraction' marketing principle, i.e.  the author’s expertise is evident, but not obtrusive.  For example, instead of writing about 'Why you need a marketing expert' or 'What a marketing expert can do for you', an article on 'Best practices for marketing’ would work better;
```

Comment:
```suggestion
  - Articles that are too advertorial in style – rather than blatantly plug the writer’s own business, articles should employ the 'attraction' marketing principle, i.e. the author’s expertise is evident, but not obtrusive. For example, instead of writing about 'Why you need a marketing expert' or 'What a marketing expert can do for you', an article on 'Best practices for marketing’ would work better;
```